docs/UserGuide.md

```diff
@@ -6,8 +6,29 @@
 If you can type fast, `FridgeFriend` can track your cold or frozen groceries faster and easier than any other apps.
 It is written in Java, and has more than 3.2kLoC.
 
-* Table of Contents
-  {:toc}
+Table of Contents
```

Comment:
Can we just name this "Contents" cos technically theres no table HAHAHAA
Also can we put a  ## heading for this?

Reply:
alright